project/password/templates/registration/password_change_complete.html

```diff
@@ -0,0 +1,8 @@
+{%extends 'base.html' %}
+
+{%block content%}
+<h2>Your password has been changed</h2>
+
+<a href="/">Profile</a>
```

Comment:
Add i18n and change link text to mention the correct destination.
```suggestion
<a href="/">
    {% translate "Go to front page" %}
</a>
```